modules/core-components/src/main/java/module-info.java

```diff
@@ -43,7 +43,9 @@
   exports org.mule.runtime.core.internal.processor to
       org.mule.runtime.deployment,
       org.mule.runtime.spring.config,
-      spring.beans;
+      spring.beans,
+      com.mulesoft.mule.runtime.kryo,
+      kryo.shaded;
```

Comment:
Do you really need this export? I believe you only need the one to `com.mulesoft.mule.runtime.kryo`

Reply:
Yes, that's right. Removed the `kryo.shaded` export.